src/services/auth.hooks.ts

```diff
@@ -29,11 +30,17 @@ export const useProvideAuth = () => {
     setUser(undefined);
   };
 
+  const updateLoadingState = () => {
+    setIsLoading(serverSignin.isLoading);
```

Comment:
You’re pulling out the correct `isLoading` variable, but I have to ask: where is the `isLoading` useState value you added being used? Your current implementation relies on the return value from calling this function. But when will the code be run again in order to get a different value from the initial call?